### PR TITLE
Compete Athena integration test

### PR DESCRIPTION
**Pull request by cw75**
## Describe your changes and why you are making these changes
Note that currently we only support reading from Athena.
I setup an Athena DB with two tables from our demo db: "customers" and "hotel_reviews", and gave everyone within our AWS org permission to access. So filling in the missing entries in `test-credentials-example.yml` should give you access to that DB. These missing fields are:`output_location` (any s3 file path that you have access to, in order to store temp results generated by Athena) and your AWS credentials. Other fields like `region` and `database` should be kept as is.

## Checklist before requesting a review
- [ ] I have created a descriptive PR title. The PR title should complete the sentence "This PR...".
- [ ] I have performed a self-review of my code.
- [ ] I have included a small demo of the changes. For the UI, this would be a screenshot or a Loom video.
- [ ] If this is a new feature, I have added unit tests and integration tests.
- [ ] I have run the integration tests locally and they are passing.
- [ ] I have run the linter script locally (See `python3 scripts/run_linters.py -h` for usage).
- [ ] All features on the UI continue to work correctly.
- [ ] Added one of the following CI labels:
    - `run_integration_test`: Runs integration tests
    - `skip_integration_test`: Skips integration tests (Should be used when changes are ONLY documentation/UI)

### CODE REVIEW

File: integration_tests/sdk/data_integration_tests/athena_test.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+def test_athena_integration_table_retrieval(client, data_integration):
+    df = data_integration.table(name="hotel_reviews")
+    assert len(df) == 100
+    assert list(df) == [
+        "hotel_name",
+        "review_date",
+        "reviewer_nationality",
+        "review",
+    ]
+
```
Comment (kenxu95) [inline integration_tests/sdk/data_integration_tests/athena_test.py:26]: Based on a quick skim through relational_test.py, I think we're just missing the case where we parameterize sql queries here? Can you also add that?

**Review by kenxu95 — APPROVED** [review]
Unblocking but please address comments.

File: integration_tests/sdk/test-credentials-example.yml [hunk 2]
```diff
@@ -32,3 +32,17 @@ data:
     # (Alternative) Config File Credentials
     # config_file_path:
     # config_file_profile:
+  test_athena:
+    type: Athena
+    database: test
+    region: us-west-1
+    # We should all have access to this bucket, so no need to change the value here.
+    output_location: s3://aqueduct-athena-temp/tmp/
```
[the remark this answers is not in the document]
Comment (cw75) [reply integration_tests/sdk/test-credentials-example.yml:40]: In terms of security I think all we're exposing is the bucket name and path, but nobody outside of our org have access to it, so I'm not super concerned here.

File: integration_tests/sdk/data_integration_tests/athena_test.py (hunk 1 above)
Comment (cw75) [reply integration_tests/sdk/data_integration_tests/athena_test.py:26]: added

**Pull request merged**